src/core/Form/Combobox/Combobox/Combobox.baseStyles.tsx

```diff
@@ -0,0 +1,51 @@
+import { css } from 'styled-components';
+import { defaultThemeTokens as theme } from '../../../theme';
+import { font } from '../../../theme/reset';
+
+export const baseStyles = css`
+  &.fi-combobox {
+    ${font(theme)('bodyText')}
+    width: 290px;
+
+    & .fi-filter-input_input-element-container {
+      position: relative;
+
+      &:before {
```

Comment:
What are these styles? I guess not possible to use existing styles?

Reply:
It is the caret icon of the input field; when list is open it will be then "upside" down.

Styles are taken from the Dropdown's caret icon with slight change (fix to pointer events). We could make it to some common, shared style?

Reply:
Ah, makes sense. I would add this to "todo" as this same icon will be needed in other components later on. Doesn't have to be done now in my opinion.

Reply:
JIRA issue is now created to our backlog from this.